Example/ViewController.swift

```diff
@@ -540,6 +540,34 @@ extension ViewController: NavigationViewControllerDelegate {
         dismissActiveNavigationViewController()
         clearNavigationMapView()
     }
+    
+    // MARK: - Offline Navigation Tiles switching
```

Comment:
Can we place it in a separate `extension ViewController: NavigationTilesVersioningDelegate`?

Reply:
Done